rule_schema_v1.yaml

```diff
@@ -953,7 +957,6 @@ properties:
               patterns: false
               pattern: false
               pattern-either: false
-              pattern-regex: false
```

Comment:
Is this related to the post-processor change?

Reply:
not quite; restored!